src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,141 @@
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// These Tooltip styles will be inlined in a Gutenberg Custom HTML block
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+///
+/// 1. Horizontally centers tooltip
+/// 2. Positions tooltip vertically in "end" position (below button)
+///
+
+.c-tooltip {
```

Comment:
Because this isn't a _full_ component yet, can we precede all these class names with an underscore [per Harry Roberts' namespacing convention](https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/#hack-namespaces-_)?